besu/src/main/java/org/hyperledger/besu/cli/options/PluginsConfigurationOptions.java

```diff
@@ -40,9 +40,8 @@ public class PluginsConfigurationOptions implements CLIOptions<PluginConfigurati
 
   @CommandLine.Option(
       names = {DEFAULT_PLUGINS_OPTION_NAME},
-      description = "Comma-separated list of plugin names",
+      description = "Comma-separated list of plugin names to load",
       split = ",",
-      hidden = true,
```

Comment:
Is it worth a changelog entry? Not sure if this option was ever mentioned before but people will see it in the help now.

Reply:
turns out we already have a changelog entry for it https://github.com/hyperledger/besu/pull/6700/files